src/v/cluster/rm_stm.cc

```diff
@@ -2359,6 +2359,10 @@ ss::future<stm_snapshot> rm_stm::take_snapshot() {
 
     for (const auto& idx : expired_abort_indexes) {
         auto filename = abort_idx_name(idx.first, idx.last);
+        vlog(
+          _ctx_log.debug,
+          "removing aborted transactions {} snapshot file",
```

Comment:
Should we also add a trace here? Makes sure it sees the file exists.

```
ss::future<> snapshot_manager::remove_snapshot(ss::sstring target) {
    if (co_await ss::file_exists(snapshot_path(target).string())) {
       ------------- here ---------------
        co_await ss::remove_file(snapshot_path(target).string());
    }

    co_return;
}

```

Reply:
good point, let me do this